.github/workflows/issue-labeled.yml

```diff
@@ -0,0 +1,40 @@
+name: Issue Labeled
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  reply-labeled:
+    runs-on: ubuntu-latest
+    steps:
+      - name: contribution welcome
+        if: github.event.label.name == 'contribution welcome' || github.event.label.name == 'help wanted'
+        uses: actions-cool/issues-helper@v2.1.1
+        with:
+          actions: 'create-comment, remove-labels'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. We totally like your proposal/feedback, welcome to send us a Pull Request for it. Please provide changelog/TypeScript/documentation/test cases if needed and make sure CI passed, we will review it soon. We appreciate your effort in advance and looking forward to your contribution!
+          labels: 'bug: pending triage, need reproduction'
+
+      - name: remove pending
+        if: github.event.label.name == 'enhancement' || github.event.label.name == 'bug: upstream' || github.event.label.name == 'bug: hmr' || github.event.label.name == 'bug'
```

Comment:
We can use a generic `label.name.startsWith('bug:')` check here in case we add more `bug: *` labels in the future.

Reply:
GitHub Action support some simple functions. See  https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#functions